HoneyBadgers.WebApp/Controllers/HomeController1.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HoneyBadgers.WebApp.Controllers
+{
+    public class HomeController1 : Controller
```

Comment:
Pusty, wygenerowany Controler?

Reply:
chyba coś się przez przypadek kliknęło :P

Reply:
Te 2 controlery to totalnie przez przypadek, wyskakiwał mi jakiś błąd jak tworzyłem kontroler :( Ale ja ich potem nigdzie nie widziałem dlatego ich nie usunąłem, dopiero teraz jak sobie w projekcie dałem Show All files to sie pojawiły. usunięte :)